apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -43,7 +43,7 @@ import { CreateNotificationTemplateQuery } from './queries';
 @Controller('/notification-templates')
 @UseInterceptors(ClassSerializerInterceptor)
 @UseGuards(JwtAuthGuard)
-@ApiTags('Workflows')
+@ApiTags('Notification Templates')
```

Comment:
Do we want to still show it in our api docs? Because 'Notification Templates' tag should be added to bootstrap. It currently shows Workflows

Reply:
Maybe we can show it for a few releases and then remove it later